gas.md

```diff
@@ -381,6 +381,8 @@ We assume that all operations interrogating the local state have complexity
   ```hs
   computationCost(LOCALCALL(_, nARGS, _) _ rARGS) =
     saveContextCost + wordCopyCost * (sum [registerSize r | r <- rARGS]) + jumpCost
+  requiredRegisterMemory(LOCALCALL(_, nARGS, _) _ rARGS) =
+    currentRegisterMemory + sum [registerSize(r) | r <- rVALUES]
```

Comment:
One question we should answer is where do we give the extra memory allowance for the call.

Reply:
Is there a memory allowance for local calls, or only for calls?

Reply:
Since this is still an open question, I added a TODO.

Reply:
After a discussion with Brandon it seems that there is a memory allowance for local calls (and local calls only), so I added a flat memory allowance per stack level, and I added some notes about other options.